deploy/addons/freshpod/freshpod-dp.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: freshpod
+  namespace: kube-system
+  labels:
+    version: v0.0.1
+    k8s-app: freshpod
+    tier: prod
+    kubernetes.io/minikube-addons: freshpod
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      version: v0.0.1
+      k8s-app: freshpod
+      tier: prod
+  template:
+    metadata:
+      labels:
+        version: v0.0.1
```

Comment:
it looks like we don't have a tagged version. I'll make sure to tag one and update the v0.0.1 image in-place.

Reply:
Okay, I have been removed from YAML file.